### DISCUSSION TITLE
There seems to be a problem with the documentation (https://pythermalcomfort.readthedocs.io/en/latest/index.html) of the package

### DISCUSSION THREAD

**Discussion by Qianjk818** [CenterForTheBuiltEnvironment/pythermalcomfort#86] [General]
Almost all of the items are only titles, the contents of which are no longer visible

**Comment by FedericoTartarini** [comment]
You are right, there was an issue with the tool that was used to generate the documentation. Now everything should be working again. Thank you very much for pointing this out to us.
